pages/sentinel_cherrypick.vue

```diff
@@ -105,15 +105,15 @@ export default {
       pickListResponse: { variant: 'danger' },
       perPage: 10,
       currentPage: 1,
-      submit_disabled: false,
+      submitDisabled: false,
```

Comment:
In sentinel_create_samples 'submit_disabled' was renamed to 'isCreating', wondering if that should be standard across these files?